### DISCUSSION TITLE
Next Image with grid-template-columns

### DISCUSSION THREAD

**Discussion by arnaudjnn** [vercel/next.js#18638] [Help]
How to use new Image component with a grid layout like this:

If I use fr it take the original image width and its no more responsive.

```jsx
.grid {
   display: flex;
   grid-template-columns: 1fr 1fr;
}

<div className="grid">
   <div>
      <Image src="/image.png" width={1000} height={1000} />
   </div>
   <div>
       <Image src="/image.png" width={1000} height={1000} />
   </div>
<div>

**Comment by Senofy** [comment]
Hey, I got the same problem, and I fixed it by using it like this
<Image src="/image.png" width={1000} height={1000} layout={'fixed'} />
